perl/README.md

```diff
@@ -38,59 +45,62 @@ The remaining dependencies are installed like this:
 cpanm --installdeps .
 ```
 
-After installing dependencies, use `prove -lr t` to run tests.
-
-## Installing Bystro locally using dzil
-
-Bystro uses [`Dist::Zilla`](https://github.com/rjbs/dist-zilla) for packaging and is configured with `dist.ini`.
-This approach requires installing `Dist::Zilla` and author dependencies and one off-dependencies described in the  above.
-
-```bash
-# Install Dist::Zilla and Archive::Tar::Wrapper (to slightly speed up building)
-cpanm --quiet Dist::Zilla Archive::Tar::Wrapper
-
-# Install build dependencies
-dzil authordeps --missing | cpanm --quiet
-
-# Install Bystro dependencies
-dzil listdeps --missing | cpanm --quiet
-
-# Install Bystro
-dzil install
-```
-
 ## Install Bystro using `cpm`
 
 Install [cpm](https://metacpan.org/pod/App::cpm) with `curl -fsSL https://raw.githubusercontent.com/skaji/cpm/main/cpm | perl - install -g App::cpm`.
 
 ```bash
 # install msgpack fork
-cpm https://github.com/bystrogenomics/msgpack-perl.git
+cpm install -g https://github.com/bystrogenomics/msgpack-perl.git
```

Comment:
we use cpm here and cpanm above; could we go to cpm throughout? I have a change I can make the corresponding change to the perlbrew script

Reply:
The problem with using `cpm` throughout is that `cpanm` is needed for installing LMDB_File - at least I couldn't find a way of using it. If your concern is that it is confusing to have two sections - one for `cpamn` and one for `cpm` - then I suggest consolidation of the two approaches with the acknowledgement that both tools are needed.